test/fixtures/anki-template-renderer-test.js

```diff
@@ -30,6 +30,7 @@ export async function createAnkiTemplateRendererTest() {
     const ankiTemplateRenderer = new AnkiTemplateRenderer();
     await ankiTemplateRenderer.prepare();
     /** @type {import('vitest').TestAPI<{window: import('jsdom').DOMWindow, ankiTemplateRenderer: AnkiTemplateRenderer}>} */
+    // eslint-disable-next-line sonarjs/prefer-immediate-return
```

Comment:
There are a few cases where I had to disable `sonarjs/prefer-immediate-return` because it needs the type hint. I could have used a cast, but a type hint is a bit safer and the syntax a bit cleaner.